lib/resources/r4/communication.yaml

```diff
@@ -114,12 +111,11 @@ fields:
   action:
     - create
 
-
 - name: encounter
   required: 'No'
   cardinality: 0..1
-  type: Reference(Encounter)
-  description: The specific Patient Encounter associated to the Communcation
+  type: Reference (Encounter)
+  description: The specific Patient Encounter associated to the Communcation.
   url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.encounter
```

Comment:
Maybe a note saying that it must be supplied with subject?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/517862a4ec5661907895442fda495fccdde5b7ce#diff-f08ae1e4c063b9a36a22adaa43fecf21R105